### PR TITLE
html-header

### PR DESCRIPTION
**Pull request by ghost**
Created html-header.

### CODE REVIEW

File: src/partials/header.html [hunk 1]
```diff
@@ -1,3 +1,140 @@
 <header class="header">
-
-</header>
\ No newline at end of file
+  <div class="header-logo-container">
+    <img
+      class="header-logo"
+      src="/src/img/logo.svg"
```
Comment (SergGrishkov) [inline src/partials/header.html]: Incorrect path. Should be change to "./img/logo.svg"

File: src/partials/header.html [hunk 2]
```diff
@@ -1,3 +1,140 @@
...
+          href="https://www.facebook.com/goITclub/"
+          target="_blank"
+        >
+          <svg
+            class="header-social-icon"
+            width="28"
+            height="28"
+            aria-label="facebook"
+          >
+            <use href="/src/img/icons.svg#icon-facebook"></use>
```
Comment (SergGrishkov) [inline src/partials/header.html]: Incorrect path. Should be change to "./img/icons.svg#icon-instagram"

File: src/partials/header.html [hunk 3]
```diff
@@ -1,3 +1,140 @@
...
+          href="https://www.instagram.com/goitclub/"
+          target="_blank"
+        >
+          <svg
+            class="header-social-icon"
+            width="28"
+            height="28"
+            aria-label="instagram"
+          >
+            <use href="/src/img/icons.svg#icon-instagram"></use>
```
Comment (SergGrishkov) [inline src/partials/header.html]: Incorrect path. Should be change to "./img/icons.svg#icon-instagram"

File: src/partials/header.html [hunk 4]
```diff
@@ -1,3 +1,140 @@
...
+          href="https://www.youtube.com/c/GoIT"
+          target="_blank"
+        >
+          <svg
+            class="header-social-icon"
+            width="28"
+            height="28"
+            aria-label="youtube"
+          >
+            <use href="/src/img/icons.svg#icon-youtube"></use>
```
Comment (SergGrishkov) [inline src/partials/header.html]: Incorrect path. Should be change to "./img/icons.svg#icon-youtube"

File: src/partials/header.html [hunk 5]
```diff
@@ -1,3 +1,140 @@
...
+  </div>
+  <div class="header-menu-container">
+    <button class="header-menu-button" type="button">
+      <svg
+        class="header-menu-open"
+        width="36"
+        height="24"
+        aria-label="menu open icon"
+      >
+        <use href="./images/icons.svg#icon-burger"></use>
```
Comment (SergGrishkov) [inline src/partials/header.html]: Incorrect path. Should be change to "./img/icons.svg#icon-burger"

File: src/partials/header.html [hunk 6]
```diff
@@ -1,3 +1,140 @@
...
+    </div>
+    <div class="menu-close-container">
+      <button class="menu-close-button" type="button">
+        <svg
+          class="menu-close"
+          width="36"
+          height="36"
+          aria-label="menu close icon"
+        >
+          <use href="./images/icons.svg#icon-close"></use>
```
Comment (SergGrishkov) [inline src/partials/header.html]: Incorrect path. Should be change to "./img/icons.svg#icon-close"

File: src/partials/header.html [hunk 7]
```diff
@@ -1,3 +1,140 @@
...
+        </li>
+        <li class="menu-navigation-item">
+          <a class="menu-navigation-link" href="">Testimonials</a>
+        </li>
+      </ul>
+    </div>
+    <div class="menu-logo-container">
+      <img
+        class="menu-logo"
+        src="/src/img/logo.svg"
```
Comment (SergGrishkov) [inline src/partials/header.html]: Incorrect path. Should be change to "./img/logo.svg"

**Review by SergGrishkov — CHANGES_REQUESTED** [review]

File: src/partials/header.html (hunk 1 above)
Comment (ghost) [reply src/partials/header.html]: Done

File: src/partials/header.html (hunk 2 above)
Comment (ghost) [reply src/partials/header.html]: Done

File: src/partials/header.html (hunk 3 above)
Comment (ghost) [reply src/partials/header.html]: Done

File: src/partials/header.html (hunk 4 above)
Comment (ghost) [reply src/partials/header.html]: Done

File: src/partials/header.html (hunk 5 above)
Comment (ghost) [reply src/partials/header.html]: Done

File: src/partials/header.html (hunk 6 above)
Comment (ghost) [reply src/partials/header.html]: Done

File: src/partials/header.html (hunk 7 above)
Comment (ghost) [reply src/partials/header.html]: Done

**Review by SergGrishkov — APPROVED** [review]

**Pull request merged**